acceptance_tests/features/pages/respondent.py

```diff
@@ -0,0 +1,28 @@
+from acceptance_tests import browser
+from config import Config
+
+
+def go_to_find_respondent():
+    browser.visit('{}/respondents/'.format(Config.RESPONSE_OPERATIONS_UI))
+
+
+def search_respondent_by_email(email):
+    browser.find_by_id('query').fill(email)
+    browser.find_by_id('btn-search-respondent').click()
+
+
+def no_results_found():
+    return 'No results found' in browser.html
+
+
+def get_respondent_details():
+    respondent_details = {
+        "name": browser.find_by_id('respondent-name').text,
+        "email": browser.find_by_id('respondent-email').text
+    }
+    return respondent_details
+
+
+def find_no_respondent_msg():
+    page_src = browser.driver.page_source
```

Comment:
Maybe the method could be no_respondent_found() and could return`'No Respondent found.' in page_src`

Something like

```
def no_respondent_found():
    return 'No Respondent found.' in browser.html
```

Reply:
and then the step would look something like
```
+@then('the internal user is given a message of no respondent for email')
def no_respondent_displayed(_):
    assert respondent.not_found()
```